src/main/java/org/broadinstitute/hellbender/tools/exome/TargetArgumentCollection.java

```diff
@@ -104,6 +127,92 @@ public TargetArgumentCollection() {
         }
     }
 
+    /**
+     * Reads a target collection from a file and adds all the annotations in additional
+     * annotation files.
+     * @param targetFile the target collection file.
+     * @param targetAnnotations additional annotation files.
+     * @return never {@code null}.
+     */
+    private TargetCollection<Target> combineTargetAnnotations(final File targetFile,
+                                                              final List<File> targetAnnotations) {
+        final TargetCollection<Target> targets = readTargetCollection(targetFile);
+
+        final Map<Target, Map<TargetAnnotation, String>> annotationsByTarget =
```

Comment:
Why? what is the advantage? Is always best to keep field and argument params as abstract as it can be. The user code does not need to relay on me using an EnumMap.